apis/nucleus/src/components/listbox/components/ListBoxSearch.jsx

```diff
@@ -175,6 +175,9 @@ export default function ListBoxSearch({
       case 'Escape': {
         focusRow(container);
         cancel();
+        if (selections.isModal()) {
+          return undefined;
+        }
```

Comment:
have you tested it in filterpane? (I think the change could cause proablem there)

Reply:
Yes I tested it with filterpane, selection.isModal() here returns true only when the listbox is a popover list.

Reply:
that sounds odd. I would have expected it to also be true when selecting in a listbox in a filterpane even when it is not a popover. (I have not tested it myself)

Reply:
@T-Wizard I haven't seen any issue yet. I uploaded a video in description part, maybe it helps to see how it works between filterpane and popoverlistbox?

Reply:
video don't play for me.

when testing the branch I can get selections.isModal() as true for inline listbox in filterpane
* click value select (not confirm)
* open search
* press escape
-> focus it lost (do work without this change)

I also get "Uncaught TypeError: hide is not a function" from filtarpane popover if closed by escape directly